primitives/src/pcs/prelude.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 Espresso Systems (espressosys.com)
+// This file is part of the Jellyfish library.
+
+// You should have received a copy of the MIT License
+// along with the Jellyfish library. If not, see <https://mit-license.org/>.
+
+//! Prelude
+pub use crate::pcs::{
+    errors::PCSError,
+    multilinear_kzg::{
+        srs::{MultilinearProverParam, MultilinearUniversalParams, MultilinearVerifierParam},
+        util::{compute_qx_degree, get_batched_nv, merge_polynomials},
+        BatchProof, KZGMultilinearPCS, Proof,
```

Comment:
`BatchProof` and `Proof` are not exactly intelligible for the downstream.

either we rename them to `MultilinearKZGBatchProof` and `MultilinearKZGProof` 
or we use alias re-export `pub use multilinear_kzg::BatchProof as MultilinearKZGBatchProof`.